src/components/UserData.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+UserData.propTypes = {
+  personalUserData: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      phone: PropTypes.string.isRequired,
+      mail: PropTypes.string.isRequired,
+      department: PropTypes.string.isRequired,
+    })
+  ),
+  skills: PropTypes.arrayOf(
+    PropTypes.shape({
+      skill1: PropTypes.string.isRequired,
+      skill2: PropTypes.string,
+      skill3: PropTypes.string,
+      skill4: PropTypes.string,
+      skill5: PropTypes.string,
+      skill6: PropTypes.string,
+    })
+  ),
+}
+
+export default function UserData({ personalUserData, skills }) {
+  return (
+    <DataWrapper>
+      <UserDates>
+        <li>Name: {personalUserData[0]}</li>
+        <li>Phone: {personalUserData[1]}</li>
+        <li>Mail: {personalUserData[2]}</li>
+        <li>Department: {personalUserData[3]}</li>
+      </UserDates>
+      <SkillsList>
+        <li>{skills[0]}</li>
+        <li>{skills[1]}</li>
+        <li>{skills[2]}</li>
+        <li>{skills[3]}</li>
+        <li>{skills[4]}</li>
+        <li>{skills[5]}</li>
```

Comment:
Please rethink how the user data is structured. This way it's not really efficient. I would probably have an object `userData` with the following structure:
```js
const userData = {
  name: 'John Doe',
  phone: 'xxx',
  email: 'xxx',
  department: 'xxx',
  skills: ['skill1', 'skill2', 'skill3']
}
```

Then you can implement it like this (also, the first user data is not really a list..):
```suggestion
        <li>Name: {userData.name}</li>
        <li>Phone: {userData.phone}</li>
        <li>Mail: {userDara.email}</li>
        <li>Department: {userData.department}</li>
      </UserDates>
      <SkillsList>
          {userData.skills.map(skill => <li key={skill}>{skill}</li>
```
This would be cleaner and a more efficient way to deal with data.